packages/passport-interface/src/PassportInterface.ts

```diff
@@ -6,6 +6,7 @@ export enum PCDRequestType {
 }
 
 export interface PCDRequest {
+  srcId: string;
```

Comment:
just wanna make sure that having a `srcId` be required is the best way to implement this

Reply:
Other option was to pass in a generic `state` variable akin to oauth, and then we pass in a callback to the popup function which will return true/false depending on if you want to accept this state, and also can do some other stuff with state. For example (state is some json object with other data as well).

What do you think?